lib/Conversion/ImportVerilog/Expressions.cpp

```
@@ -791,6 +791,49 @@ struct RvalueExprVisitor {
     return visitAssignmentPattern(expr, *count);
   }
 
+  Value visit(const slang::ast::StreamingConcatenationExpression &expr) {
+    SmallVector<Value> operands;
+    for (auto stream: expr.streams()) {
+      auto value = context.convertRvalueExpression(*stream.operand);
+      if (!value)
+        continue;
+      value = context.convertToSimpleBitVector(value);
+      if(!value) {
+        mlir::emitError(loc) << "Streaming concatentation do only support SimpleBitVector for now";
+        // return {};
```

The `return {};` would be good to have 👍 Fail loud, reliably, and early 😃